### PR TITLE
feat(impact): improve styling of services impact section

### PR DESCRIPTION
**Pull request by kjwessa**
### TL;DR

This pull request implements significant refactoring changes to the Services page components and their styling. 

Key changes include:

1. **ServiceCard:** Fixed the prop name from `href` to `link`.
2. **ServicesImpact:** Simplified the component by replacing div elements with the new `ImpactCard` component.
3. **ServicesIntro:** Adjusted padding for better visual alignment.
4. **ServicesROI:** Improved nesting and formatting within the JSX structure.
5. **ServicesSpeciality:** Removed unused border classes for a cleaner design.
6. **ServicesTestimonial:** Removed the top border for a more modern look.
7. **page.tsx:** Included `LineFull` component to add horizontal lines separating sections.
8. **ImpactCard Component:** Introduced a new `ImpactCard` reusable component for displaying impact metrics.
9. **LineFull Component:** Introduced a new `LineFull` component for adding horizontal lines.
10. **LinkMain Component:** Changed the prop name from `href` to `link` for consistency.

---

### CODE REVIEW

**Comment by kjwessa** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/kjwessa/brewww-studio/119?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#119** <a href="https://app.graphite.dev/github/pr/kjwessa/brewww-studio/119?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#118** <a href="https://app.graphite.dev/github/pr/kjwessa/brewww-studio/118?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#116** <a href="https://app.graphite.dev/github/pr/kjwessa/brewww-studio/116?utm_source=stack-comment-icon" target="_blank">Graphite</a>: 1 other dependent PR ([#117](https://github.com/kjwessa/brewww-studio/pull/117) <a href="https://app.graphite.dev/github/pr/kjwessa/brewww-studio/117?utm_source=stack-comment-icon" target="_blank">Graphite</a>)
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kjwessa and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

**Comment by kjwessa** [comment]
### Merge activity

* **Aug 7, 11:49 PM EDT**: @kjwessa started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/kjwessa/brewww-studio/119).
